### PR TITLE
Add  rand int

### PR DESCRIPTION
**Pull request by kochurovro**

### CODE REVIEW

**Comment by mholt** [comment]
Nice! Does rand get seeded anywhere? I don't see any call to `rand.Seed`: https://github.com/Masterminds/sprig/search?q=seed&unscoped_q=seed

**Comment by kochurovro** [comment]
Oh. Yep. I forgot about it. going to update it

File: functions.go [hunk 1]
```diff
@@ -199,7 +199,10 @@ var genericMap = map[string]interface{}{
 		}
 		return val
 	},
-	"randInt": func(min, max int) int { return rand.Intn(max-min) + min },
+	"randInt": func(min, max int) int {
+		rand.Seed(time.Now().UnixNano())
```
Comment (mholt) [inline functions.go]: This should be done only once, preferably:

```go
func init() {
    rand.Seed(time.Now().UnixNano())
}
```

**Comment by kochurovro** [comment]
@mholt what u think about my solution for TestShuffle?

**Comment by kochurovro** [comment]
@mholt what u think about my solution for TestShuffle? Is it correct for test?

**Comment by mholt** [comment]
I don't really know why it had to change, but at a cursory glance it looks fine, as the tests appear to be passing, so, that's a good sign.

**Comment by Thiht** [comment]
Hi, any news on this PR? :)

File: docs/math.md [hunk 2]
```diff
@@ -69,3 +69,10 @@ Returns the greatest float value greater than or equal to input value
 Returns a float value with the remainder rounded to the given number to digits after the decimal point.
 
 `round 123.555555 3` will return `123.556`
+
+## randInt
+Returns a random integer value from min (inclusive) to max (exclusive).
+```
+{{ randInt 12 30 }}
+```
+The above will produce a random number in the range [12,30).
```
Comment (danielhoherd) [inline docs/math.md:78]:
```suggestion
The above will produce a random number in the range [12,30].
```

**Review by technosophos — APPROVED** [review]

File: docs/math.md (hunk 2 above)
Comment (francislavoie) [reply docs/math.md:78]: That notation is correct: https://en.wikipedia.org/wiki/Interval_(mathematics)#Integer_intervals

Comment (danielhoherd) [reply docs/math.md:78]: TIL

**Pull request merged**